Tests/SentryTests/Integrations/Feedback/SentryFeedbackTests.swift

```diff
@@ -43,6 +43,39 @@ class SentryFeedbackTests: XCTestCase {
         XCTAssertEqual(try XCTUnwrap(attachments.first).contentType, "application/png")
     }
     
+    func testSerializeCustomFeedback() throws {
+        let sut = SentryFeedback(message: "Test feedback message", name: "Test feedback provider", email: "test-feedback-provider@sentry.io", source: .custom, attachments: [Data()])
+        
+        let serialization = sut.serialize()
+        XCTAssertEqual(try XCTUnwrap(serialization["message"] as? String), "Test feedback message")
```

Comment:
`l`: Why the nesting of `XCTUnwrap` and `XCTAssertEqual`? AFAIK the XCTUnwrap is unnecessary, because we can compare `serialization["message"] as? String` directly to `Test feedback message`

Reply:
Here's a comparison of how they fail:
```
func testFoo() throws {
    let a: String? = nil
    XCTAssertEqual(a, "a")                // 1. XCTAssertEqual failed: ("nil") is not equal to ("Optional("a")")
    XCTAssertEqual(try XCTUnwrap(a), "a") // 2. XCTUnwrap failed: expected non-nil value of type "String"
}
```
I prefer (2) because (1) coerces even the string literal in the expression to an optional type, which is not the intended comparison. I want to both ensure it's not nil, and then compare then concrete value inside it. I prefer it failing at the unwrap before trying to compare.